cron/centreon-send-stats.php

```diff
@@ -121,7 +121,7 @@ function logger(string $message, Throwable $exception = null)
          * Only send statistics if user using a free version has enabled this option
          * or if at least a Centreon license is valid
          */
-        if ($shouldSendStatistics || $hasValidLicenses || $isRemote) {
+        if ($shouldSendStatistics || $hasValidLicenses) {
```

Comment:
need to check with PM why we send stats when option is disables and valid licenses are present

Reply:
It's a normal usecase, paid licenses implicitely enable send stats